### PR TITLE
[OSPP] The proposal for pedestrian tracking and Re-ID example.

### PR DESCRIPTION
**Pull request by yqhok1**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
proposal for Implement function of automatically algorithm performance tests for Sedna Edge-cloud Re-ID

**Which issue(s) this PR fixes**:

### CODE REVIEW

**Comment by JimmyYang20** [comment]
@yqhok1 The length of the pr title and commit can not exceed 50 characters.

**Comment by JimmyYang20** [comment]
@yqhok1 
The commit signature is missing.
git commit -s -m "xxxx"
refer to [https://github.com/kubeedge/ianvs/pull/20/checks?check_run_id=7790016720](https://github.com/kubeedge/ianvs/pull/20/checks?check_run_id=7790016720)

File: docs/proposals/test-reports/pedestrian_tracking.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Implement function of automatically algorithm performance tests for Sedna Edge-cloud Re-ID
+
```
Comment (jaypume) [inline docs/proposals/test-reports/pedestrian_tracking.md:2]: The title "Motivation" should be added here.

File: docs/proposals/test-reports/pedestrian_tracking.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+Person Re-ID based on Cloud-edge collaborative architecture is a new example feature that Sedna will soon support, which can continuously recognize, track and search the target person provided by the user in the source video, and push the video containing the search results to the streaming media server. Since each service module flexibly supports the AI model, it is necessary to provide users with the actual performance test of THE AI algorithm to provide a reference for the actual search effect of the system.
+
+## Goals
+
+1. Supports the existing capabilities and runtime of Sedna and integrate into the Re-ID features;
+2. Use open-source data sets to create test reports automatically, need to define metrics, related interface, preprocessing, and etc;
+3. Have detailed guidance documents.
```
Comment (jaypume) [inline docs/proposals/test-reports/pedestrian_tracking.md:9]: It is suggested to add a complete subject here, such as "This framework provides xxx", "This feature supports xxx"

File: docs/proposals/test-reports/pedestrian_tracking.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+8. **Rank-20**: top 20 hit rate
+
+Reference
+
+[1] Liang Zheng, Liyue Shen, Lu Tian, Shengjin Wang, Jingdong Wang, Qi Tian; Proceedings of the IEEE International Conference on Computer Vision (ICCV), 2015, pp. 1116-1124
+
+[2] Wei Li, Rui Zhao, Tong Xiao, Xiaogang Wang; Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition (CVPR), 2014, pp. 152-159
+
+### Framework
+![framework](images/pedestrian_tracking_framework.png)
```
Comment (MooreZheng) [inline docs/proposals/test-reports/pedestrian_tracking.md:88]: As mentioned in the previous meeting, Re-ID shall not belong to single-task leanring. It should be of multi-edge inference. Thus the framework figure should be revised accordingly.

File: docs/proposals/test-reports/pedestrian_tracking.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    |-- pedestrian re-identification
+|- datasets
+    |-- pedestrian detection & tracking
+    |-- pedestrian re-identification
+|- metrics
+    |-- pedestrian detection & tracking
+    |-- pedestrian re-identification
+|- usage
+|- reports
+```
```
Comment (MooreZheng) [inline docs/proposals/test-reports/pedestrian_tracking.md:114]: Re-ID Benchmarking files

ianvs
|- example
    |- re-id
        |- multi-edge inference bench
            |- testalgorithm
                |- mynet.yaml (named as the used algorithm)
            |- testenv
                |- testenv.yaml
                |- mAP.py (named as the used metric)
            |- benchmarking.yaml

Comment (MooreZheng) [reply docs/proposals/test-reports/pedestrian_tracking.md:127]: looks fine to me now

File: docs/proposals/test-reports/pedestrian_tracking.md (hunk 3 above)
Comment (MooreZheng) [reply docs/proposals/test-reports/pedestrian_tracking.md:88]: The comment has revised as suggested

File: docs/proposals/test-reports/pedestrian_tracking.md (hunk 1 above)
Comment (MooreZheng) [reply docs/proposals/test-reports/pedestrian_tracking.md:2]: The comment has been revised

**Pull request merged**